src/app/clusters/door-lock-server/door-lock-server-user.cpp

```diff
@@ -620,7 +623,7 @@ static bool verifyPin(uint8_t * pin, uint8_t * userId)
     return false;
 }
 
-bool emberAfDoorLockClusterLockDoorCallback(EndpointId endpoint, app::CommandHandler * commandObj, uint8_t * PIN)
+bool emberAfDoorLockClusterLockDoorCallback(EndpointId endpoint, app::CommandHandler * commandObj, chip::ByteSpan PIN)
```

Comment:
Why is PIN even CHAR_STRING?  It's octstr in the spec...  Followup issue to fix that?

Reply:
I think octstr was not supported at the time.